docs/walkthrough.md

```diff
@@ -0,0 +1,231 @@
+
+```bash
+# Start server container in the background
+docker run -d --gpus=all --network=host -v $PWD:/mnt --name triton-server nvcr.io/nvidia/tritonserver:22.07-py3 
+
+# Start SDK container in the background
+docker run -d --gpus=all --network=host -v $PWD:/mnt --name triton-sdk nvcr.io/nvidia/tritonserver:22.07-py3-sdk
+```
+
+
+
+4. Serve the model with Triton
+
+To serve our model, we will use the `server` container that we already started which comes pre-installed with a `tritonserver` binary.
+
+```bash
+# Enter server container interactively
+docker exec -ti triton-server bash
+
+# Start serving your models
+tritonserver --model-repository=/mnt/models
+```
+
+> NOTE: The `-v $PWD:/mnt` is mounting your current directory on the host into the `/mnt` directory inside the container. So if you created your model repository in `$PWD/models`, you will find it inside the container at `/mnt/models`; You can change these paths as needed. See [docker volume](https://docs.docker.com/storage/volumes/) docs for more information on how this works.
+
+5. Verify the model can run inference
+
+To verify our model can perform inference, we will use the `triton-sdk` (client) container that we already started which comes with `perf_analyzer`  pre-installed.
+
+In a separate shell, we use Perf Analyzer to sanity check that we can run inference and get a baseline for the kind of performance we expect from this model.
+
+In the example below, Perf Analyzer is sending requests to models served on the same machine (`localhost` from the server container via `--network=host`). However, you may also test models being served remotely at some `<IP>:<PORT>`  by setting the `-u` flag, such as `perf_analyzer -m alexnet -u 127.0.0.1:8000`.
+
+```bash
+# Enter SDK container interactively
+docker exec -ti triton-sdk bash
+
+# Benchmark model being served from step 3
+perf_analyzer -m alexnet --concurrency-range 1:4
+```
+
+```
+...
+Inferences/Second vs. Client Average Batch Latency
+Concurrency: 1, throughput: 265.147 infer/sec, latency 3769 usec
+Concurrency: 2, throughput: 890.793 infer/sec, latency 2243 usec
+Concurrency: 3, throughput: 937.036 infer/sec, latency 3199 usec
+Concurrency: 4, throughput: 965.21 infer/sec, latency 4142 usec
+```
+
+6. Run Model Analyzer to find the best configurations for our model
+
+While Model Analyzer comes pre-installed in the SDK container and supports various modes of connecting to a Triton server,
+for simplicity we will use install Model Analyzer in our `server` container to use the `--triton-launch-mode=local` flag.
+To learn more about other methods of connecting Model Analyzer to a running Triton Server, see the `--triton-launch-mode`
+Model Analyzer flag.
+
+```bash
+# Enter server container interactively
+docker exec -ti triton-server bash
+
+# Stop existing tritonserver process if still running
+# because model-analyzer will start its own server
```

Comment:
Nit: consistency of spelling (comment below says "model analyzer", one later in the document also says "model-analyzer")